README.md

```diff
@@ -364,10 +364,11 @@ only enabled with the `--ignoreJWTSignature` flag. Make sure to only set this fl
 
 You are able to use some custom headers in your request to gain more control over the requestContext object.
 
-| Header                          | Event key                                                   |
-| ------------------------------- | ----------------------------------------------------------- |
-| cognito-identity-id             | event.requestContext.identity.cognitoIdentityId             |
-| cognito-authentication-provider | event.requestContext.identity.cognitoAuthenticationProvider |
+| Header                          | Event key                                                   | Example |
+| ------------------------------- | ----------------------------------------------------------- | ------- |
+| cognito-identity-id             | event.requestContext.identity.cognitoIdentityId             |         |
+| cognito-authentication-provider | event.requestContext.identity.cognitoAuthenticationProvider |         |
+| SLS_OFFLINE_AUTHORIZER_OVERRIDE | event.requestContext.authorizer                             | { "iam": {"cognitoUser": { "amr": ["unauthenticated"], "identityId": "abc123" }}} |
```

Comment:
maybe we should use standard header-naming pattern here instead of one that looks like env variable?

Reply:
no problem, i've updated the PR